pandas-stubs/io/pickle.pyi

```diff
@@ -10,5 +10,5 @@ def to_pickle(
 ): ...
 def read_pickle(
     filepath_or_buffer_or_reader: FilePathOrBuffer,
-    compression: str | Literal["infer", "gzip", "bz2", "zip", "xz"] | None = ...,
+    compression: Literal["infer", "gzip", "bz2", "zip", "xz"] | None = ...,
```

Comment:
Should be using `CompressionOptions` here (and probably elsewhere you fixed `compression` as an argument)